beacon-chain/blockchain/receive_block.go

```diff
@@ -26,6 +26,7 @@ type BlockReceiver interface {
 	ReceiveBlock(ctx context.Context, block interfaces.ReadOnlySignedBeaconBlock, blockRoot [32]byte) error
 	ReceiveBlockBatch(ctx context.Context, blocks []interfaces.ReadOnlySignedBeaconBlock, blkRoots [][32]byte) error
 	HasBlock(ctx context.Context, root [32]byte) bool
+	GetResentBlockSlot(root [32]byte) (primitives.Slot, error)
```

Comment:
What's the purpose of using the word `Resent` here? Was it supposed to be `Recent`?

Reply:
Renamed it to `ResentBlockSlot` :)